.eslintrc.js

```diff
@@ -22,7 +22,33 @@ module.exports = {
         "@typescript-eslint/no-inferrable-types": ["warn", { ignoreProperties: true }],
         "@typescript-eslint/no-unused-vars": ["warn", { varsIgnorePattern: "^(?:h|Component|Element|Prop|Listen|Watch|Method|State|Event)$" }],
         "@typescript-eslint/explicit-module-boundary-types": ["warn", { allowedNames: ["render"] }],
-        "@stencil/async-methods": "off",
+        "@stencil-community/async-methods": "off",
+        "@stencil-community/decorators-context": "error",
+        "@stencil-community/decorators-style": [
+            "error", {
+            "prop": "inline",
+            "state": "inline",
+            "element": "inline",
+            "event": "inline",
+            "method": "multiline",
+            "watch": "multiline",
+            "listen": "multiline"
+            }
+        ],
+        "@stencil-community/element-type": "error",
+        "@stencil-community/methods-must-be-public": "error",
+        "@stencil-community/no-unused-watch": "error",
+        "@stencil-community/own-methods-must-be-private": "error",
+        "@stencil-community/own-props-must-be-private": "error",
+        "@stencil-community/prefer-vdom-listener": "error",
+        "@stencil-community/props-must-be-public": "error",
```

Comment:
I think we have some internal props like: https://github.com/mi6/ic-ui-kit/blob/main/packages/web-components/src/components/ic-breadcrumb/ic-breadcrumb.tsx#L37

I'm assuming I've misunderstood this lint rule.

Reply:
So that lint rule checks whether props have the `private` or `protected` keyword next to it. Seems to be a strange rule however because I just tried it and I couldn't even build the ui-kit because it raised an error, so I feel as though you'd spot this before running the linter.